### PR TITLE
Add CNAlifestyle handler

### PR DESCRIPTION
**Pull request by changhuapeng**
Tests on CNAlifestyle URLs:
- https://www.reddit.com/r/rsgretrivr/comments/auaenc/sometimes_parenting_is_boring_then_you_feel/eh6r00y/
- https://www.reddit.com/r/rsgretrivr/comments/au8967/as_trump_and_kim_prepare_for_summit_bars_in/eh6hpmd/
- https://www.reddit.com/r/rsgretrivr/comments/auainv/roses_leopard_prints_and_lots_of_lace_at/eh6qzyg/

Tests for HandlerManager in detecting non-CNAlifestyle URLs:
- https://www.reddit.com/r/rsgretrivr/comments/au8czo/huawei_chairman_says_trumps_latest_message_on_5g/eh6hpkf/
- https://www.reddit.com/r/rsgretrivr/comments/au8fyt/exodus_from_last_isis_enclave_overwhelms_syria/eh6hpio/
- https://www.reddit.com/r/rsgretrivr/comments/au8pf8/what_will_the_budget_package_look_like_when/eh6hpda/
- https://www.reddit.com/r/rsgretrivr/comments/au8sse/8_hard_truths_about_jc_in_singapore_i_learnt/eh6hpc6/
---
Each domain names will have a default handler now. For instance, _ArticleHandler_ is declared in the lambda function as the default one for _channelnewsasia.com_.

&nbsp;
**In the event when subdomain handler is declared**
URL: _https://cnalifestyle.channelnewsasia.com/travel/trump-kim-summit-vietnam-bars-cocktails-11276220_

Since _cnalifestyle_ subdomain has been declared, _CNAlifestyleHandler_ will be triggered.

&nbsp;
**In the event when subdomain handler is not declared**
URL: _https://www.channelnewsasia.com/news/business/huawei-chairman-says-trump-s-latest-message-on-5g--clear-and-correct--11284730_
or
_https://channelnewsasia.com/news/business/huawei-chairman-says-trump-s-latest-message-on-5g--clear-and-correct--11284730_

Since there are no declared _www_ subdomain or that no subdomain is provided in the URL, the default handler _ArticleHandler_ will be triggered.

Fix for #12

### CODE REVIEW

**Comment by yleong** [comment]
I think this looks good and can be merged unless anyone objects.

Extracting article body via p tags with no bad children is a great idea. I think something similar can be done to improve the other handlers that just naively use `get_text()`

File: handlers/CNAlifestyleHandler/__init__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
...0%20%20%7D%0A%20%20%20%20defaultPhoto%20%7B%0A%20%20%20%20%20%20link%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20photos%20%7B%0A%20%20%20%20%20%20teaserText%0A%20%20%20%20%20%20detailText%0A%20%20%20%20%20%20photo%20%7B%0A%20%20%20%20%20%20%20%20link%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20video%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20ooyalaId%0A%20%20%20%20%20%20duration%0A%20%20%20%20%20%20playerId%0A%20%20%20%20%20%20category%0A%20%20%20%20%20%20sk%20%7B%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20ner%20%7B%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20audio%20%7B%0A%20%20%20%20%20%20duration%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20html%0A%20%20%20%20teaserText%0A%20%20%20%20contentLength%0A%20%20%20%20postedDate%0A%20%20%20%20__typename%0A%20%20%7D%0A%7D%0A&operationName=article&variables=%7B%22categoryId%22%3A%22{category_id}%22%2C%22articleId%22%3A%22{article_id}%22%7D"
```
Comment (xlanor) [inline handlers/CNAlifestyleHandler/__init__.py]: Can this be separated into multiline f-strings to align with PEP-8? I understand this is the nature of the URL, but Im thinking if multi-line f-strings would look neater in this case

Comment (changhuapeng) [reply handlers/CNAlifestyleHandler/__init__.py]: I agreed and have separated that into multi-line f-strings.

Comment (xlanor) [reply handlers/CNAlifestyleHandler/__init__.py]: Thank you!

If @fterh has no objections, I will merge this into develop.

**Comment by xlanor** [comment]
I spoke to Fabian and he's okay with the changes.

Merging this into develop for the next release.

**Pull request merged**